src/enums/allowableActions.ts

```diff
@@ -1,24 +1,25 @@
+/** Enum used internally to query allowable actions via mixin. */
 export enum AllowableActions {
-  ADDRESS_CHANGE = 'addressChange',
-  ADMINISTRATIVE_DISSOLUTION = 'administrativeDissolution',
-  ANNUAL_REPORT = 'annualReport',
-  BUSINESS_INFORMATION = 'businessInformation',
-  BUSINESS_SUMMARY = 'businessSummary',
-  CONSENT_CONTINUATION_OUT = 'consentContinuationOut',
-  CORRECTION = 'correction',
-  COURT_ORDER = 'courtOrder',
-  DETAIL_COMMENT = 'detailComment',
-  DIGITAL_CREDENTIALS = 'digitalCredentials',
-  DIRECTOR_CHANGE = 'directorChange',
-  FREEZE_UNFREEZE = 'freezeUnfreeze',
-  LIMITED_RESTO_CONVERT = 'limitedRestoConvert',
-  LIMITED_RESTO_EXTEND = 'limitedRestoExtend',
-  PUT_BACK_ON = 'putBackOn',
-  RECORD_CONVERSION = 'recordConversion',
-  REGISTRARS_NOTATION = 'registrarsNotation',
-  REGISTRARS_ORDER = 'registrarsOrder',
-  RESTORATION = 'restoration',
-  STAFF_COMMENT = 'staffComment',
-  TRANSITION = 'transition',
-  VOLUNTARY_DISSOLUTION = 'voluntaryDissolution',
+  ADDRESS_CHANGE = 1, // skip 0 which is falsy
+  ADMINISTRATIVE_DISSOLUTION,
+  ANNUAL_REPORT,
+  BUSINESS_INFORMATION,
+  BUSINESS_SUMMARY,
+  CONSENT_CONTINUATION_OUT,
+  CORRECTION,
+  COURT_ORDER,
+  DETAIL_COMMENT,
+  DIGITAL_CREDENTIALS,
+  DIRECTOR_CHANGE,
+  FREEZE_UNFREEZE,
+  LIMITED_RESTORATION_EXTENSION,
+  LIMITED_RESTORATION_TO_FULL,
+  PUT_BACK_ON,
+  RECORD_CONVERSION,
+  REGISTRARS_NOTATION,
+  REGISTRARS_ORDER,
+  RESTORATION,
+  STAFF_COMMENT,
+  TRANSITION,
+  VOLUNTARY_DISSOLUTION,
```

Comment:
The above creates an enum with values 1 through 22.

(The initial value is optional. If I left it out then values would be 0-relative.)